lib/ansible/modules/database/proxysql/proxysql_query_rules.py

```diff
@@ -513,16 +541,20 @@ def main():
             match_digest=dict(type='str'),
             match_pattern=dict(type='str'),
             negate_match_pattern=dict(type='bool'),
+            re_modifiers=dict(type='str'),
             flagOUT=dict(type='int'),
             replace_pattern=dict(type='str'),
             destination_hostgroup=dict(type='int'),
             cache_ttl=dict(type='int'),
             timeout=dict(type='int'),
             retries=dict(type='int'),
             delay=dict(type='int'),
+            next_query_flagIN=dict(type='int'),
             mirror_flagOUT=dict(type='int'),
             mirror_hostgroup=dict(type='int'),
             error_msg=dict(type='str'),
+            OK_msg=dict(type='str'),
+            multiplex=dict(choices=[0, 1, 2]),
```

Comment:
I got the error msg `"value of multiplex must be one of: 0, 1, 2, got: 1"`
The type `type='int'` is missing for multiplex.

Reply:
thanks for testing :) I added the type attribute